src/ts/components/volumeslider.ts

```diff
@@ -1,5 +1,6 @@
 import {SeekBar, SeekBarConfig} from './seekbar';
 import {UIInstanceManager} from '../uimanager';
+import { VolumeController, VolumeTransition } from '../volumecontroller';
```

Comment:
I guess the `VolumeController` import is not needed here. Also not imported in `volumetogglebutton.ts`